app/(internal)/profile/page.tsx

```
@@ -0,0 +1,1495 @@
+        : new Date().toISOString(),
+      reward: 'Beta tester badge',
+    },
+    {
+      id: 'one-year-member',
+      name: 'Veteran Member',
+      description: 'Member for over one year',
+      icon: Award,
+      category: 'milestone',
+      rarity: 'uncommon',
+      unlocked:
+        user && user.createdAt
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)